src/mongosh/schemas/curriculum-2.0.0.json

```diff
@@ -0,0 +1,99 @@
+{
+	"description": "A Learning resource or other activity that is on a person's career or learning roadmap",
+	"bsonType": "object",
+	"required": [
+		"_id",
+		"resources"
+	],
+	"properties": {
+		"_id": {
+			"description": "The unique identifier for a resource used by a person",
+			"bsonType": "objectId"
+		},
+		"version": {
+			"description": "Schema Version (only on Version document)",
+			"bsonType": "string",
+			"maxLength": 20
+		},
+		"resources": {
```

Comment:
you will need to add a "name" property temporarily - just a type:string - we will remove this in a future iteration, so declare it but don't use it.